client/my-sites/plans-features-main/components/free-plan-paid-domain-dialog.tsx

```diff
@@ -119,30 +121,82 @@ const StyledButton = styled( Button )`
 	}
 `;
 
-export function FreePlanPaidDomainDialog( {
+function GetSuggestedPlanSection( {
 	domainName,
 	suggestedPlanSlug,
-	onFreePlanSelected,
-	onPlanSelected,
-	onClose,
+	onButtonClick,
+	isBusy,
 }: {
 	domainName: string;
 	suggestedPlanSlug: PlanSlug;
-	onClose: () => void;
-	onFreePlanSelected: ( domainSuggestion: DomainSuggestion ) => void;
-	onPlanSelected: () => void;
+	onButtonClick: () => void;
+	isBusy: boolean;
 } ) {
 	const translate = useTranslate();
-	const queryClient = useQueryClient();
-	const [ isBusy, setIsBusy ] = useState( false );
 	const planPrices = usePlanPrices( { planSlug: suggestedPlanSlug, returnMonthly: true } );
 	const currencyCode = useSelector( getCurrentUserCurrencyCode );
+	const planTitle = getPlan( suggestedPlanSlug )?.getTitle();
+
+	return (
+		<>
+			<DomainName>
+				<div>{ domainName }</div>
+				<FreeDomainText>{ translate( 'Free for one year ' ) }</FreeDomainText>
```

Comment:
Just noticed that there was an extra space here. (Was present earlier too)
```suggestion
				<FreeDomainText>{ translate( 'Free for one year' ) }</FreeDomainText>
```

Reply:
Good catch! I've committed the fix :)